app/controllers/status_controller.rb

```diff
@@ -1,5 +0,0 @@
-class StatusController < ApplicationController
-  def index
-    render json: { alive: true }
```

Comment:
This endpoint is used as a probe by kubernetes so it should be left in place. (see https://github.com/ministryofjustice/check-financial-eligibility/blob/master/deploy/helm/templates/deployment_web.yaml#L33)

Reply:
It seems to be 100% now 🙂